library/infoblox_host_record.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+# Copyright (c) 2017 Philip Bove
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = """
+module: infoblox_host_record
+description: manage infoblox host records with infoblox-client
+version_added: "2.4"
+"""
+
+EXAMPLES = """
+- name: create host_record
+  host: "{{grid_manager}}"
+  name: server.test.com
+  mac: macaddr
+  ip_address: 10.1.10.50
+  username: test
+  password: test
+"""
+from ansible.module_utils.basic import *
+
+try:
+	from infoblox_client import objects, connector, exceptions
+except ImportError:
+	raise Exception('infoblox-client is not installed.  Please see details here: https://github.com/infobloxopen/infoblox-client')
```

Comment:
I need to fix the other module, but the preferred method in ansible core at least is to assign a value here, and exit with `fail_json`

e.g. https://github.com/ansible/ansible/blob/d8b1cb9a634d0e828a7be39d4019a6215410e643/lib/ansible/modules/network/panos/panos_commit.py#L105-L106

Reply:
Not a problem.